src/promptflow/promptflow/_sdk/data/executable/README.md

```diff
@@ -0,0 +1,13 @@
+Exported entry file & its dependencies are located in the same folder. The structure is as below:
+- flow: the folder contains all the flow files
+  - ...
+- connections: the folder contains yaml files to create all related connections
+  - ...
+- app.py: the entry file is included as the entry point for the bundled application.
+- app.spec: the spec file tells PyInstaller how to process your script.
+- main.py: it will start streamlit service and be called by the entry file.
+- settings.json: a json file to store the settings of the executable application.
+- README.md: the readme file to describe how to use the exported files and scripts.
+
+Please refer to [official doc](https://microsoft.github.io/promptflow/how-to-guides/deploy-a-flow/index.html)
+for more details about how to use the exported files and scripts.
```

Comment:
Will [azure] impact final build app size?
![image](https://github.com/microsoft/promptflow/assets/47586720/a392c0e9-d9f2-4384-a4a3-76dc56613d61)

Reply:
It depends on whether your virtual env install azure. 

If it has installed, pyinstaller will collect azure relevant hidden imports (but its size is small compared with hidden imports and dlls when using streamlit)

Reply:
offline synced, keep this for now